### ISSUE TITLE
Doesn't handle nested aliases?

### ISSUE THREAD

**Issue by brian428** [Rush/link-module-alias#9]
I can't seem to create a nested alias. For example, using:

```
"_moduleAliases": {
  "@company/some-app": "./some/path"
},
```
Throws an error. But just using:
```
"_moduleAliases": {
  "@some-app": "./some/path"
},
```
Will run successfully. Ideally, link-module-alias would create the necessary directories under node_modules and then add the symlink in for the last part of the alias. Hopefully this makes sense. And if I'm missing something or doing something wrong here, please let me know.

**Comment by aggarwaldev** [comment]
+1

**Comment by creativenoobie** [comment]
@Rush Any thoughts on this?

**Comment by Rush** [comment]
@creativenoobie thanks for pinging - I will take a look at this

**Comment by kwburnett** [comment]
@Rush I created a PR for this issue because I needed the fix for some work I'm doing. The change is working well and doing what I need it to do, so please review and let me know feedback when you've got a chance.

**Comment by Rush** [comment]
Fixed by #11. Appreciate the PR!
